jakarta.batch.arquillian.exec.example/README.md

```diff
@@ -0,0 +1,40 @@
+<!--- 
+Copyright (c) 2021 Contributors to the Eclipse Foundation
+
+See the NOTICE file distributed with this work for additional information regarding copyright 
+ownership. Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. You may 
+obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 
+Unless required by applicable law or agreed to in writing, software distributed 
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES 
+OR CONDITIONS OF ANY KIND, either express or implied. See the License for 
+the specific language governing permissions and limitations under the License. 
+SPDX-License-Identifier: Apache-2.0
+--->
+
+# An example how to run the TCK with the Arquillian execution module
+
+## How to run the TCK against GlassFish
```

Comment:
Assume this isn't actually possible with a publicly-available build still ?

Does the TCK test DB need to be configured?

Reply:
Now, the public GF build doesn't pass the TCK. We'll have to wait until GF team adds support for the current spec requirements.